src/PrettyPrinter.ml

```diff
@@ -91,6 +118,16 @@ module Doc : DOC = struct
                ( letbox_kwd
                ^^^ !^(Id.M.to_string idm)
                ^^^ equals ^^^ walk 2 boxed ^^^ in_kwd ^/^ walk 1 body ))
+      | Match { matched; zbranch; pred; sbranch } ->
+          (parens_if (p > 1))
+            ( match_kwd ^^^ walk 1 matched ^^^ with_kwd ^/^ bar ^^^ zero_kwd
+            ^^^ darrow
+            ^^^ nest 10 (walk 1 zbranch)
```

Comment:
Let's define the constant `10` via the lengths of `zero_kwd`, double arrow and throw in the length of the whitespace.

Reply:
fixed